### DISCUSSION TITLE
Jekyll_app1htmlFile/jekyll/onefileapp/2020/09/18/Skryptozakladki

### DISCUSSION THREAD

**Discussion by giscus** [andrzejQ/Jekyll_app1htmlFile#14] [Komentarze]
# Jekyll_app1htmlFile/jekyll/onefileapp/2020/09/18/Skryptozakladki

Skryptozakładki to małe 1-wierszowe aplikacje Javascript, które można umieszczać w pasku zakładek swojej przeglądarki internetowej - przeciągając wybraną skryptozakładkę do paska zakładek (wideo).

http://localhost:4000/Jekyll_app1htmlFile/jekyll/onefileapp/2020/09/18/Skryptozakladki.html

**Comment by andrzejQ** [comment]
Nowa wersja komentarzy z użyciem `giscus -> GitHub Discussions`.
